modules/dhcp_libvirt/subnet_service_initializer.rb

```diff
@@ -37,7 +37,7 @@ def parse_config_for_subnets
       raise Proxy::DHCP::Error("Only one subnet is supported") if ret_val.size > 1
       ret_val
     rescue Exception => e
-      msg = "Unable to parse subnets XML"
+      msg = "Unable to parse subnets XML: #{e}"
       logger.error msg, e
```

Comment:
Isn't the issue than actually that the logger.error ignores the second parameter and other places are also affected?

Reply:
Yeah I was thinking the same this morning.